tests/test_xopen.py

```diff
@@ -566,3 +566,67 @@ def test_xopen_zst_fails_when_zstandard_not_available(monkeypatch):
     with pytest.raises(ImportError):
         with xopen.xopen(TEST_DIR / "file.txt.zst", mode="rb", threads=0) as f:
             f.read()
+
+
+# tests for passing in filehandles
+def test_passing_in_gz_filehandles_for_reading():
```

Comment:
The standard term to use would be ["file object"](https://docs.python.org/3/glossary.html#term-file-object) or file-like object instead of handle.